index.html

```diff
@@ -360,6 +401,25 @@ <h3>
                     The <dfn>errorDescription</dfn> attribute is a developer-friendly textual description of the [=Document/error=] [=Document/global application lifecycle states=].
                 </p>
             </section>
+            
+            <section>
+                <h3>
+                    `lang` attribute
+                </h3>
+                <p>
+                    The <dfn>lang</dfn> attribute specifies the primary language for the values of the errorDescription attribute.
```

Comment:
Here it should say that the value of this attribute is a BCP 47 language tag.

Reply:
Do you mean I need specifically say in the spec that this attribute is a BCP 47 language tag? Or only change to `referrerInfo` and `errorDescription` is good enough?

Reply:
> Do you mean I need specifically say in the spec that this attribute is a BCP 47 language tag?

Yes. It should say something like this:

> The value of the `lang` attribute MUST be a well-formed language tag as defined by [[BCP47](https://www.rfc-editor.org/info/bcp47)].

Reply:
`errorDescription` should be changed as well.

Reply:
> `errorDescription` should be changed as well.

Updated